Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1450,3 +1450,10 @@ def stripGrid(vtk_grid):
             thresh.Update()
             vtk_grid = thresh.GetOutput()
     return vtk_grid
+
+def vtkIterate(iterator):
+    iterator.InitTraversal()
```

Comment:
This was just a useful function that I kept needing at various points, so I stuck it into vcs2vtk for easy access.